src/static/css/almanac.css

```diff
@@ -389,6 +389,35 @@ h2.header {
   text-align: center;
 }
 
+.btn.chapter-cta > svg {
+  margin: 0px 5px;
+}
+
+.chapter-cta {
+  position: fixed;
+  border: none;
+  box-shadow: 2px 2px 3px 3px lightgray;
+  padding: 10px 15px;
+  opacity: 0.9;
+  text-decoration: none;
+  z-index: 1;
+}
+
+.comment-cta {
```

Comment:
Rather than positioning each CTA individually, I'd recommend wrapping them all in a `<div>` and positioning that div on the bottom right. Then the CTAs within the div could be positioned relative to each other according to flexbox.

Something like this (not tested):

```css
#cta-container {
  position: fixed;
  bottom: 0.6em;
  right: 0.125em;
  display: flex;
  flex-direction: column;
}

.chapter-cta {
  margin-top: 1em;
}
```